src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 900;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
```

Comment:
let's use this advice https://codeguide.academy/html-css.html#css-order

Reply:
Ok. Thanks for usefull link. But i am a little bit confused, because i've read this rules https://9elements.com/css-rule-order/ and there are some different rules from your link, for example that border's styles we have to put infront of font's.

Reply:
Ok. I undertood what you meant)